libs/langchain/langchain/document_loaders/kaggle_dataset.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Iterator, List
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+
+class KaggleDatasetLoader(BaseLoader):
+    """Load from `Kaggle` datasets.
+
+    Follow these steps to use this loader:
+    - Register a Kaggle account and create an API token to use this loader.
```

Comment:
I haven't used the kaggle package before, and would it be possible to automate these steps in the loader instead of having it be a csv loader with some logic around the kaggle dataset format?

Reply:
It is possible. But I don't have time for it right now. I'm switching this PR to draft. Maybe I'll find time for automate it.